### PR TITLE
Faction specific jobs

### PR DESCRIPTION
**Pull request by Cdey78**
This took WAY longer than needed.
--
Adds faction-specific jobs to the game.
Each job MUST have a unique name, for future reference, because the job system won't have it any other way.

:cl: optional name here
add: Added faction specific jobs
add: This means each and every job has it's own faction, aswell as some factions having jobs that others don't.
/:cl:

### CODE REVIEW

**Comment by Cdey78** [comment]
Spotted a few hiccups

**Comment by Kmc2000** [comment]
m a d l a d

File: StarTrek13/Ships/factions.dm [hunk 1]
```diff
@@ -98,14 +100,20 @@ var/global/list/factionRosters[][] = list(list("Independent Roster"),
 	name = "nanotrasen"
 	description = "Nanotrasen, or more specifically, their main corporate arm. Their goal is to make money and maintain the colonies, no matter who opposes them."
 	flavourtext = "Nanotrasen is an oligarchy, but with merit you should be able to climb the ranks...up to a point."
-	pref_colour = "blue"
+	pref_folour = "blue"
```
Comment (nichlas0010) [inline StarTrek13/Ships/factions.dm]: 😕

Comment (Kmc2000) [reply StarTrek13/Ships/factions.dm]: So that's why travis was yelling

Comment (Cdey78) [reply StarTrek13/Ships/factions.dm]: OOF

**Pull request merged**